multimodal/src/autogluon/multimodal/models/utils.py

```diff
@@ -684,3 +684,19 @@ def update_mmdet_config(key, value, config):
             for subsubconfig in subconfig:
                 if isinstance(subsubconfig, dict):
                     update_mmdet_config(key, value, subsubconfig)
+
+
+def run_model(model: nn.Module, batch: dict):
+    if type(model).__module__.endswith("timm_image"):
```

Comment:
Wondering if all backbones in TIMM will be ONNX compatible or just the ones we use as of now